independent-publisher-connectors/snowflake/apiDefinition.swagger.json

```diff
@@ -0,0 +1,745 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Snowflake REST",

```

Comment:
@rekodus Can you please remove the word called "REST" from the title? It should be the name of your product or organization. Please follow the instructions mentioned in the certification documentation.

https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title